htdocs/xoops_lib/modules/protector/library/HTMLPurifier/HTMLModule/TargetNoreferrer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Module adds the target-based noreferrer attribute transformation to a tags.  It
+ * is enabled by HTML.TargetNoreferrer
+ */
+class HTMLPurifier_HTMLModule_TargetNoreferrer extends HTMLPurifier_HTMLModule
+{
+    /**
+     * @type string
+     */
+    public $name = 'TargetNoreferrer';
+
+    /**
+     * @param HTMLPurifier_Config $config
+     */
+    public function setup($config) {
+        $a = $this->addBlankElement('a');
```

Comment:
__[phpmd]__
* Avoid variables with short names like $a. Configured minimum length is 3.

<!-- Comment by [SideCI](https://sideci.com) -->